tests/test_s3_bucket_operations.py

```diff
@@ -0,0 +1,189 @@
+import logging
+import os
+from datetime import datetime, timedelta, timezone
+
+import pytest
+
+from noobaa_sa.exceptions import BucketAlreadyExists, BucketNotEmpty, NoSuchBucket
+
+log = logging.getLogger(__name__)
+
+
+class TestS3BucketOperations:
+    """
+    Test S3 bucket operations using NSFS
+    """
+
+    def test_bucket_creation_deletion_and_head(self, c_scope_s3client):
+        """
+        Test bucket creation and deletion via S3:
+        1. Create a bucket via S3
+        2. Verify the bucket was created via S3 HeadBucket
+        3. Delete the bucket via S3
+        4. Verify the bucket was deleted via S3 HeadBucket
+
+        """
+        # 1. Create a bucket via S3
+        bucket_name = c_scope_s3client.create_bucket()
+        # 2. Verify the bucket was created via S3 HeadBucket
+        assert c_scope_s3client.head_bucket(bucket_name), "Bucket was not created"
```

Comment:
Please check the response code to validate bucket creation

Reply:
Done in https://github.com/red-hat-storage/noobaa-sa-ci/pull/16/commits/027ec7aac1aaa31c2625b0649ff6bf65372da000